devbootcamps/website
# Ticket Orders & Payments

In order to make this run, you need a couple of ENV vars:

```ruby
ENV['DB_STRIPE_PKEY'] # A public Stripe key
ENV['DB_MOLLY_KEY']   # An API key for your Mollie account
```

Take these steps to test it:

- Start your local server and go to [`/tickets`](http://localhost:3000/tickets)
- Order a couple of tickets, try with one and with more tickets
- Read all texts thoroughly and check them for errors/inconsistencies/etc.
- Make a payment through Mollie
- Make a payment through Stripe


Note that Mollie's and Stripe's webhooks do not work on your local env.

#### Before deploy

- [x] Add Stripe key to production env
- [x] Add Mollie key to production env
- [ ] Make sure migrations are ran
